src/main/resources/assets/waila/lang/de_DE.lang

```diff
@@ -69,16 +77,14 @@ option.vanilla.leverstate=Hebelstatus
 option.vanilla.repeater=Repeater-Verzögerung
 option.vanilla.comparator=Komparatormodus
 option.vanilla.redstone=Redstone-Stärke
-option.vanilla.alternatecropitem=Wechselnde Ernte
-option.vanilla.jukebox=Musikbox-Platte
+option.vanilla.jukebox=Plattenspieler-Schallplatte
 option.vanilla.silverfish=Silberfisch verdecken
-option.vanilla.furnacedisplay=Zeige Ofen Fortschritt
+option.vanilla.furnacedisplay=Zeige den Fortschritt des Ofens
```

Comment:
Having the article makes the options sound clunky. I'd keep them removed.

Reply:
I think it would be better to remove the first article than both.

Reply:
Yes, of course.